core-bundle/src/Migration/Version500/RssToFeedReaderMigration.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Contao.
+ *
+ * (c) Leo Feyer
+ *
+ * @license LGPL-3.0-or-later
+ */
+
+namespace Contao\CoreBundle\Migration\Version500;
+
+use Contao\CoreBundle\Migration\AbstractMigration;
+use Contao\CoreBundle\Migration\MigrationResult;
+use Doctrine\DBAL\Connection;
+
+class RssToFeedReaderMigration extends AbstractMigration
```

Comment:
Feels like this whole migration could be simplified by renaming the columns if the old ones exist and the new ones do not?

Reply:
Fixed